bool Init_start

---

bool Init_done
